docs/src/intro.md

```diff
@@ -125,3 +126,21 @@ docker-compose up
 ```
 
 *Quality-time* is served at [http://localhost](http://localhost). Use username `jadoe` and password `secret` to log in.
+
+# Why *Quality-time*?
+
+ICTU developed *Quality-time* to help projects and teams within ICTU gain actionable insight into the quality of the software they are developing and maintaining. The current vision on the scope of *Quality-time* is:
+
+- *Quality-time* collects data from sources that projects and teams are using anyway to develop and maintain software. *Quality-time* does not analyse the software itself.
+- The sources that *Quality-time* uses to collect its measurement data are automated. Manual sources for measurements are possible, but automated sources are preferred.
+- *Quality-time* evaluates measurement data against target values. It shows users which metrics do not meet their target value, so users can take corrective action. *Informational* metrics, meaning metrics without target, are possible, but most metrics are assumed to have targets.
+- *Quality-time* provides a single overview of the software measurements, but refers users to the underlying tools for detailed information.
+- Technical debt management is an integral part of software quality management. Any metric that does not meet its target value for a longer period of time can be considered to represent a form of debt that needs to be managed.
+- Tools come and go. Hence, an important non-functional requirement for *Quality-time* is the ability to quickly add interfacing to new sources. However, it is not a requirement to make this possible without developing a small amount of interface code.
+- *Quality-time* provides actionable information on current quality issues and risks. Historical information is retained, but is a second-class citizen.
+- *Quality-time* is not an issue tracker or task manager. However, it does integrate with issue trackers so it's easy to create issues for metrics that need action and to manage technical debt.
```

Comment:
Easier reading:
`It does however integrate with issue trackers, so`

Reply:
Solved differently.